packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -53,6 +54,13 @@ export const DefaultCell = (props: TableCellProps) => {
 
   if (isStringValue) {
     let justifyContent = cellProps.style?.justifyContent;
+    const str = value.toString();
+    if (
+      ((str.startsWith('{') && str.endsWith('}')) || (str.startsWith('[') && str.startsWith(']'))) &&
```

Comment:
@leeoniya -- any thoughts here?  with a regexp is this reasonable?

Reply:
i'd just do  `str[0] === '{' || str[0] === '['`. we don't need to check closing braces, since an actual parse try/catch will tell us if it's properly formed anyhow.